modules/cli/src/integrationTest/java/org/apache/ignite/cli/HandlerAdapter.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli;
+
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+/**
+ * Adapter for {@link Handler} with empty implementations of all methods but {@link Handler#publish(LogRecord)}.
+ */
+public abstract class HandlerAdapter extends Handler {
```

Comment:
The value of this class is to make the 'use site' code cleaner. Two more empty methods would distract the reader.

I understand that `HandlerAdapter` makes you think it could be an instance of Adapter design pattern and, hence, it could contain some non-trivial logic. I renamed it to `NoOpHandler` to highlight that it is just an empty implementation.

What do you think?